src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackServiceTest.java

```diff
@@ -194,4 +291,10 @@ void should_report_errors_when_ocr_data_is_invalid() {
             .asString()
             .matches(match);
     }
+
+    private void setUpTransformationUrl() {
+        ServiceConfigItem configItem = new ServiceConfigItem();
+        configItem.setTransformationUrl("url");
```

Comment:
Doesn't matter but it would be nice to have at least correct url format `http://servicehost/transform-exception-record`

Reply:
yes, it absolutely doesn't matter